DataFormats/Math/test/BuildFile.xml

```diff
@@ -69,3 +69,18 @@
 <bin name="testDataFormatsMathPacking" file="testMiniFloat.cpp,testlogintpack.cpp,testRunner.cpp">
   <use   name="cppunit"/>
 </bin>
+
+<use name="cuda"/>
+<use name="cuda-api-wrappers"/>
+<use   name="vdt_headers"/>
```

Comment:
IIUC these should be inside the individual `<bin>...</bin>` elements below to not make the other binaries depend on cuda (or vdt).